src/shared/components/NcTile/NcTile.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<div class="tile" :class="{active: localeActive}" @click="$emit('set-template')">
+		<h3>{{ title }}</h3>
+		<p>{{ body }}</p>
+	</div>
+</template>
+<script>
+
+export default {
+	components: {
+	},
+
+	filters: {
+		truncate(string, num) {
+			if (string.length >= num) {
+				return string.substring(0, num) + '...'
+			} else {
+				return string
+			}
+		},
+	},
+
+	mixins: [],
+
+	props: {
+		active: {
+		      type: Boolean,
+		      default: false,
+		    },
+		title: {
+		      type: String,
+		      default: '',
+		    },
+		body: {
+		      type: String,
+		      default: '',
+		    },
```

Comment:
Tabs instead of spaces and the closing bracket aligned with the opening line:

```suggestion
			type: Boolean,
			default: false,
		},
		title: {
			type: String,
			default: '',
		},
		body: {
			type: String,
			default: '',
		},
```